### DISCUSSION TITLE
how can i use data from global Data by javascript code

### DISCUSSION THREAD

**Discussion by toilagakfc** [AutomaApp/automa#216] [Q&A]
hi, i try to get value in globaldata/datacolumns by javascript code,but idk how to do this. Please help me and thanks for your help!

![Capture](https://user-images.githubusercontent.com/78404799/147273806-15ce6ce4-1d4e-440b-9493-8ec004669c8a.PNG)
this is my code in javascript

![image](https://user-images.githubusercontent.com/78404799/147273986-25a189a0-d4dc-4ffa-b095-837e05630bf0.png)
this is my global data

**Comment by Kholid060** [comment]
```js
const text = automaRefData('globalData', '0.city')
```
